src/tracing/mux.rs

```diff
@@ -21,10 +25,13 @@ use revm::{
 use thiserror::Error;
 
 /// Mux tracing inspector that runs and collects results of multiple inspectors at once.
-#[derive(Clone, Debug)]
+#[derive(Debug)]
+#[cfg(feature = "js-tracer")]
```

Comment:
since the fields are private we can simply feature gate the addiiotnal jsInspector here